tapdb/universe_stats.go

```diff
@@ -152,7 +153,7 @@ type assetEventsCache = *lru.Cache[eventQuery, cachedAssetEvents]
 
 // statsQueryCacheSize is the total number of asset query responses that we'll
 // hold inside the cache.
-const statsQueryCacheSize = 80_0000
```

Comment:
TIL: you can place underscore separators anywhere

Reply:
Yep, super useful for numbers like this! In this case I still miscounted the amount of zeros lol.